lib/BUILD

```diff
@@ -1,7 +1,7 @@
 cc_library(
```

Comment:
BUILD changes to master have been covered in this PR https://github.com/googleinterns/scsi2nvme/pull/83